CONTRIBUTING.md

```diff
@@ -14,3 +14,223 @@ If you are sending a pull request and it's a non-trivial change beyond fixing ty
 [the ICLA (individual contributor license agreement)](https://feedback.line.me/enquete/public/799-Z6ygVz3U). Please
 contact us if you need the CCLA (corporate contributor license agreement).
 
+### How to build
```

Comment:
Agreed, but I just don't want to put duplicate information. i.e. 'How to build' in README.md and CONTRIBUTING.md. Perhaps I could add a link to CONTRIBUTING.md in README.md?